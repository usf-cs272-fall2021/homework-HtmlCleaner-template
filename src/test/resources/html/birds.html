
<!DOCTYPE html>

<html lang="en">
<head>
<title>Web Crawler Test</title>
<meta charset="utf-8">
</head>

<body>

<p>Here is a list of birds:</p>

<ul>
	<li><a href="albatross.html">albatross</a></li>
	<li><a href="blackbird.html">blackbird</a></li>
	<li><a href="bluebird.html">bluebird</a></li>
	<li><a href="cardinal.html">cardinal</a></li>
	<li><a href="chickadee.html">chickadee</a></li>
	<li><a href="crane.html">crane</a></li>
	<li><a href="crow.html">crow</a></li>
	<li><a href="cuckoo.html">cuckoo</a></li>
	<li><a href="dove.html">dove</a></li>
	<li><a href="duck.html">duck</a></li>
	<li><a href="eagle.html">eagle</a></li>
	<li><a href="egret.html">egret</a></li>
	<li><a href="falcon.html">falcon</a></li>
	<li><a href="finch.html">finch</a></li>
	<li><a href="goose.html">goose</a></li>
	<li><a href="gull.html">gull</a></li>
	<li><a href="hawk.html">hawk</a></li>
	<li><a href="heron.html">heron</a></li>
	<li><a href="hummingbird.html">hummingbird</a></li>
	<li><a href="ibis.html">ibis</a></li>
	<li><a href="kingfisher.html">kingfisher</a></li>
	<li><a href="loon.html">loon</a></li>
	<li><a href="magpie.html">magpie</a></li>
	<li><a href="mallard.html">mallard</a></li>
	<li><a href="meadowlark.html">meadowlark</a></li>
	<li><a href="mockingbird.html">mockingbird</a></li>
	<li><a href="nighthawk.html">nighthawk</a></li>
	<li><a href="osprey.html">osprey</a></li>
	<li><a href="owl.html">owl</a></li>
	<li><a href="pelican.html">pelican</a></li>
	<li><a href="pheasant.html">pheasant</a></li>
	<li><a href="pigeon.html">pigeon</a></li>
	<li><a href="puffin.html">puffin</a></li>
	<li><a href="quail.html">quail</a></li>
	<li><a href="raven.html">raven</a></li>
	<li><a href="roadrunner.html">roadrunner</a></li>
	<li><a href="robin.html">robin</a></li>
	<li><a href="sandpiper.html">sandpiper</a></li>
	<li><a href="sparrow.html">sparrow</a></li>
	<li><a href="starling.html">starling</a></li>
	<li><a href="stork.html">stork</a></li>
	<li><a href="swallow.html">swallow</a></li>
	<li><a href="swan.html">swan</a></li>
	<li><a href="tern.html">tern</a></li>
	<li><a href="turkey.html">turkey</a></li>
	<li><a href="vulture.html">vulture</a></li>
	<li><a href="warbler.html">warbler</a></li>
	<li><a href="woodpecker.html">woodpecker</a></li>
	<li><a href="wren.html">wren</a></li>
	<li><a href="yellowthroat.html">yellowthroat</a></li>
</ul>

<p>
<a href="birds.html">Home</a>
</p>

</body>
</html>
