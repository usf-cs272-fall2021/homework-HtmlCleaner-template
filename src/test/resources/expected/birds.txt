







Here is a list of birds:


	albatross
	blackbird
	bluebird
	cardinal
	chickadee
	crane
	crow
	cuckoo
	dove
	duck
	eagle
	egret
	falcon
	finch
	goose
	gull
	hawk
	heron
	hummingbird
	ibis
	kingfisher
	loon
	magpie
	mallard
	meadowlark
	mockingbird
	nighthawk
	osprey
	owl
	pelican
	pheasant
	pigeon
	puffin
	quail
	raven
	roadrunner
	robin
	sandpiper
	sparrow
	starling
	stork
	swallow
	swan
	tern
	turkey
	vulture
	warbler
	woodpecker
	wren
	yellowthroat



Home




